### PR TITLE
update kubectl output in examples

### LINKED ISSUE #10168: Update all kubectl get output in the documentation
As it is happening at #10167, we need to update all the documentation on the `kubectl get` output. Also inform the user for the `--label-column=` option for further information.

This is due to the update we had in #9032 and #9876

### PR DESCRIPTION
**Pull request by caesarxuchao**
Update the rest examples.
Fixes #10168
@JanetKuo @krousey @nikhiljindal

### CODE REVIEW

File: examples/spark/README.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ $ kubectl create -f examples/spark/spark-worker-controller.json
 
 ```shell
 $ kubectl get pods
-NAME                                           READY     REASON    RESTARTS   AGE
+NAME                                            READY     REASON    RESTARTS   AGE
```
Comment (JanetKuo) [inline examples/spark/README.md]: "REASON" should be "STATUS"

Comment (caesarxuchao) [inline examples/spark/README.md]: Thanks. I have another PR that changes all "REASON" to "STATUS". I expect to see a merge conflicts here :)

File: examples/storm/README.md [hunk 2]
```diff
@@ -52,15 +52,15 @@ before proceeding.
...
+NAME        READY     STATUS    RESTARTS   AGE
+zookeeper   1/1       Running   0          43s
 ```
 
 ### Check to see if ZooKeeper is accessible
 
 ```shell
 $ kubectl get services
-NAME                LABELS                                    SELECTOR            IP                  PORT
+NAME                LABELS                                    SELECTOR            IP(S)                  PORT
```
Comment (JanetKuo) [inline examples/storm/README.md]: PORT(S)

File: examples/storm/README.md [hunk 3]
```diff
@@ -94,7 +94,7 @@ Ensure that the Nimbus service is running and functional.
 
 ```shell
 $ kubectl get services
-NAME                LABELS                                    SELECTOR            IP                  PORT
+NAME                LABELS                                    SELECTOR            IP(S)               PORT
```
Comment (JanetKuo) [inline examples/storm/README.md]: PORT(S)

**Comment by caesarxuchao** [comment]
Thank you @JanetKuo! I've addressed the comments in another commit. I will squash them when I get the LGTM.

**Comment by JanetKuo** [comment]
LGTM thanks

**Comment by rjnagal** [comment]
@caesarxuchao this is ready to merge. Can you squash commits?

**Comment by caesarxuchao** [comment]
Squashed. Thank you @rjnagal !

**Pull request merged**